cypress.json

```
@@ -1,4 +1,6 @@
 {
-	"video": false,
-	"baseUrl": "http://localhost:7080"
-}
+  "video": false,
+  "env": {
+    "APP_ENV": "local"
+  }
+}
```

```suggestion
}

```
Just needs an EOF blank line